swarm/tools/flow_studio_ui/fragments/65-context-budget-modal.html

```
@@ -61,13 +61,13 @@ <h4>Profile Override</h4>
     <div class="settings-presets">
       <h4>Quick Presets</h4>
       <div class="preset-buttons">
-        <button class="btn-preset" data-preset="lean" title="Minimal history for fast execution">
+        <button class="btn-preset" data-preset="lean" title="Minimal history for fast execution" aria-label="Apply Lean Preset">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)